### ISSUE TITLE
Design and build new Directory of Meetings page, incl. menu link

### ISSUE THREAD

**Issue by brylie** [WesternFriend/WF-website#560]
Edit: The current Wagtail site allows the content editor to build and edith the new Directory of Meetings page. So, we are limited in what we can do in advance, mainly since the yearly meetings are database entities that will be imported from the old site.

So, this issue could be repurposed to train @marycklein about how to build the custom Directory of Meetings page.

We are renaming the Community page to be called Directory of Meetings. We need to agree on the design and functionality within this project's scope.

Task
- [x] agree on the main components for the Directory of Meetings page
- [ ] train @marycklein how to build page layout using Wagtail blocks
- [ ] create custom blocks as needed
- [x] include page in navigation menu scaffold

**Comment by brylie** [comment]
@marycklein, where should we link to the Community landing page in the new navigation menu? On the sandbox, we have a link to "Directories" under "About Us." Do we want to add the "Community" landing page link under "About Us" or somewhere else? 

For reference, I am referring to what we call the "Community" page on the existing Western Friend website: https://westernfriend.org/community

**Comment by marycklein** [comment]
A "landing page" for "community" will no longer be needed in the new menu structure. The current Drupal "community" page leads to four main "sub-landing pages" which will be access it wagtail thus - 
1 - "Events" (currently nested under "welcome") - will be in drop-downs under the main menu bar in wagtail
2 - "Past annual sessions" (currently nested under "welcome") - we can provide a link to this collection in the "landing page" for each annual meeting as well as in our media library
3 - "Western Friend Directory" - my understanding is that we are doing our best to replicate this in wagtail; it will be in a drop-down under the main nav bar
4 - "Western Friend Connect" - in drupal, this leads to a "sub-landing page" that offers content that I think is actually mostely obsolete; let's talk about what I need to do to clean this up in Drupal

Bottom line: Items 1 & 3 above will be accessed in wagtail menu via sketch I'm attaching here. I've highlighted in yellow the connections. Items 2 & 4 -- We should talk about where to put links to this content and how I might make those changes in Drupal "now."


![Image](https://user-images.githubusercontent.com/44241543/234071229-8492c9d7-6a8e-46d9-8c92-e196748e7718.png)

**Comment by brylie** [comment]
> "Western Friend Directory" - my understanding is that we are doing our best to replicate this in wagtail; it will be in a drop-down under the main nav bar

So, will the "Directory of Meetings" list Yearly Meetings, behaving like the Western Friend Directory section of the existing Community page?

The last time we agreed on a menu structure, there was a "Directories" link under About Us, which was somewhat ambiguous.

https://github.com/WesternFriend/WF-website/issues/552

Since I am maintaining the agreed menu structure in code, please make menu change requests directly on GitHub issues rather than a spreadsheet to keep us in sync.

**Comment by brylie** [comment]
Also, aside from the custom text and links you've added manually, I've been using the existing Community page structure as a reference when building the new site.  As a next step, we need to clarify the design you'd like for the new Community page, even if we now refer to it as a Directory of Meetings.

<img width="1124" alt="Screenshot 2023-04-25 at 8 33 22" src="https://user-images.githubusercontent.com/17307/234183422-7868fb44-9ab0-43e4-ae9e-dc746a00aac4.png">

**Comment by marycklein** [comment]
Hi Brylie -

progress progress !!

Yes re . . . old “Western Friend Directory section of community page” = New “Directory of Meetings”

And yes, you’re right — That page will want to be designed. I think probably it will pretty much only contain three links that will go to the three directory pages for the three yearly meetings. 

Maybe “directory” page could also include a link to a listing of other quaker organizations, which could combine what we’ve now got listed under “quaker nonprofit organizations” and “Friends everywhere”

We do not need to show link to online worship here. It fits more logically as an item under “events” in nav bar.

Again, let’s talk about what I can do to clean some of this up in Drupal “now.” — combine the all the “quaker organization” listings into one page now maybe; as well as clean up stuff on that “western friend connect” page

!!
m



Mary Klein
Editor / Executive Director

Western Friend
2625 Middlefield Road, #565
Palo Alto, CA  94306-2516
650-318-1652

**Comment by marycklein** [comment]
I have removed obsolete material from "community" page on Drupal.

**Comment by brylie** [comment]
Cool beans. I've updated this issue description to note that it is now a training issue about constructing the Directory of Meetings using the new Wagtail site.
